app/views/katello/api/v2/systems/base.json.rabl

```diff
@@ -0,0 +1,12 @@
+object @resource
+
+@resource ||= @object
+
+node(:id) { |resource| resource.uuid }
+attributes :id => :katello_id
+attributes :uuid, :name, :description
+attributes :location
+attributes :content_view, :content_view_id
+attributes :distribution
+attributes :katello_agent_installed? => :katello_agent_installed
+attributes :registered_by
```

Comment:
Looks like some needed fields are missing in the base rabl:

lifecycle environment, registered (time), last checkin, errata counts

In the UI these items are blank